pkg/minikube/constants/constants.go

```diff
@@ -426,3 +426,8 @@ const (
 	// DriverDocumentation the documentation of the KVM driver
 	DriverDocumentation = "https://github.com/kubernetes/minikube/blob/master/docs/drivers.md"
 )
+
+const (
+	CACertificatesDir = "/usr/share/ca-certificates"
+	SSLCertStoreDir   = "/etc/ssl/certs"
```

Comment:
Could these be moved to `bootstrapper/certs.go`? I prefer to keep constants as close to their user as possible, unless they must be shared by multiple packages.